initialize/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM quay.io/cybozu/moco-mysql:8.0.20.4
 ENV GOPATH=/go
 ENV PATH=/go/bin:/usr/local/go/bin:"$PATH"
 
-ARG GO_VERSION=1.13.15
+ARG GO_VERSION=1.15.7
```

Comment:
This enlarges the final image.
Why don't you split the build stage?

Reply:
This Dockerfile is used for the unit test of the `initialize` package.
The test is launched as follows (execute `go test` in the container), so it needs to be installed.

https://github.com/cybozu-go/moco/blob/v0.3.1/Makefile#L46
```
docker run -v $(PWD):/go/src/github.com/cybozu-go/moco -e GOPATH=/tmp --rm mysql-with-go:latest sh -c "CGO_ENABLED=0 go test -v ./initialize"
```

Reply:
I see.